python/dolfinx/fem/__init__.py

```diff
@@ -34,6 +46,30 @@ def create_sparsity_pattern(a: Form):
     return _create_sparsity_pattern(a._cpp_object)
 
 
+def create_nonmatching_meshes_interpolation_data(
+    mesh_to: typing.Union[_Mesh, typing.Union[_Geometry_float64, _Geometry_float32]],
+    element: typing.Union[_FiniteElement_float32, _FiniteElement_float64],
+    mesh_from: _Mesh, cells: typing.Optional[npt.NDArray[np.int32]] = None, padding: float = 1e-14
+):
+    """Generate data needed to interpolate discrete functions across different meshes.
+
+    Args:
+        mesh_to: Mesh or geometry to interpolate into
```

Comment:
Interpolation into a function space makes sense, but it's not clear to me what interpolation in a mesh or geometry means.

Reply:
I've attempted to make this clearer (based on my incomplete understanding of what the C++ function is doing)